tests/ClientTest.php

```diff
@@ -750,7 +750,7 @@ public function testExceptionOnInvalidIdn()
         $client = new Client(['handler' => $mockHandler]);
 
         $this->expectException(\GuzzleHttp\Exception\InvalidArgumentException::class);
-        $this->expectExceptionMessage('IDN conversion failed (errors: IDNA_ERROR_LEADING_HYPHEN)');
```

Comment:
Could you remind me why this is needed?

Reply:
Because the polyfill has a different error message sometimes, and I think also the extension depending on how it was compiled. We are only interested in checking that IDN conversion failed, and not the exact string of the message.